config/prod/neo4j2owl_config_schema.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "neo4j2owl-config schema",
+  "description": "A JSON schema for documenting and validating the neo4j2owl configuration YAML.",
+  "name": "neo4j2owl_config_schema",
+  "type": "object",
+  "additionalProperties": false,
+  "definitions": {
+    "semantic_tag": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "Specifies a set of Manchester syntax class expressions and a label. Annotates all entities that are equivalent_to or subclasses of these expression with the label using the AP: {TBA} .  In conversion to Neo4J these become neo4j:labels",
+      "properties": {
+        "classes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "A list of Manchester syntax class expressions in which OWL entities are expressed as curies."
```

Comment:
Only cosmetic, but 'CURIEs' is written with capital letters in documentation: https://www.w3.org/2001/sw/BestPractices/HTML/CURIE